lib/node_modules/@stdlib/blas/ext/base/scusumpw/src/addon.c

```diff
@@ -0,0 +1,56 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2020 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "stdlib/blas/ext/base/scusumpw.h"
+#include "stdlib/napi/export.h"
+#include "stdlib/napi/argv.h"
+#include "stdlib/napi/argv_float.h"
+#include "stdlib/napi/argv_int64.h"
+#include "stdlib/napi/argv_strided_float32array.h"
+#include <node_api.h>
+
+
+/**
+* Computes the cumulative sum of single-precision floating-point strided array elements using pairwise summation.
+*
+* ## Notes
+*
+* -   When called from JavaScript, the function expects six arguments:
+*
+*     -   `N`: number of indexed elements
+*     -   `sum`: initial sum
+*     -   `X`: input array
+*     -   `strideX`: `X` stride length
+*     -   `Y`: output array
+*     -   `strideY`: `Y` stride length
+*/
+static napi_value addon( napi_env env, napi_callback_info info ) {
+	STDLIB_NAPI_ARGV( env, info, argv, argc, 6 );
+	STDLIB_NAPI_ARGV_INT64( env, N, argv, 0 );
+	STDLIB_NAPI_ARGV_FLOAT( env, sum, argv, 1 );
+	STDLIB_NAPI_ARGV_INT64( env, strideX, argv, 3 );
+	STDLIB_NAPI_ARGV_INT64( env, strideY, argv, 5 );
+	STDLIB_NAPI_ARGV_STRIDED_FLOAT32ARRAY( env, X, N, strideX, argv, 2 );
+	STDLIB_NAPI_ARGV_STRIDED_FLOAT32ARRAY( env, Y, N, strideY, argv, 4 );
+
+	stdlib_strided_scusumpw( N, (float)sum, (float *)X, strideX, (float *)Y, strideY );
```

Comment:
Casting is already taken care of now with the macros:

```suggestion
	stdlib_strided_scusumpw( N, sum, X, strideX, Y, strideY );
```